src/chains/ethereum/options/src/logging-options.ts

```diff
@@ -110,5 +123,23 @@ export const LoggingOptions: Definitions<LoggingConfig> = {
     legacyName: "verbose",
     cliAliases: ["v", "verbose"],
     cliType: "boolean"
+  },
+  file: {
+    normalize: rawInput => {
+      // this will throw if the file is not writable, and creates the log file for later appending
+      try {
+        const descriptor = openSync(rawInput, "w");
```

Comment:
This will truncate the file if it exists. Is that the desired behavior? flag `wx` would fail if the file exists, or, `a` for appending, but won't truncate if the file exists.

Reply:
'a' is the desired behaviour - thanks for pointing that out!

Reply:
I'll add a test for that too